sockeye/inference.py

```diff
@@ -854,12 +854,21 @@ class LengthPenalty:
     :param beta: The beta factor for the length penalty (see above).
     """
 
-    def __init__(self, alpha: float = 1.0, beta: float = 0.0) -> None:
+    def __init__(self, alpha: float = 1.0, beta: float = 0.0, **kwargs) -> None:
+        super().__init__(**kwargs)
         self.alpha = alpha
         self.beta = beta
         self.denominator = (self.beta + 1.) ** self.alpha
 
-    def __call__(self, lengths: Union[mx.nd.NDArray, int, float]) -> Union[mx.nd.NDArray, float]:
+    def hybrid_forward(self, F, lengths):
+        if self.alpha == 0.0:
+            return F.ones_like(lengths)
+        else:
+            numerator = self.beta + lengths if self.beta != 0.0 else lengths
+            numerator = numerator ** self.alpha if self.alpha != 1.0 else numerator
+            return numerator / self.denominator
+
+    def get(self, lengths: Union[mx.nd.NDArray, int, float]) -> Union[mx.nd.NDArray, float]:
```

Comment:
Could we merge `get` and `hybrid_forward` somehow?  e.g.: 
```
   def get(...):
       return hybird_forward(None, lengths)
   def hybrid_forward(F, lengths):
        if F is None:
             return 1
         else:
             return F.ones_like(...)
```

Reply:
sure, done